src/routes.tsx

```diff
@@ -304,6 +304,7 @@ export const routes: Route[] = [
                   return {
                     baseUrl: `/${locale}/new-member/home-accident-switcher`,
                     name: EmbarkStory.SwedenSwitcher,
+                    quoteCart: true,
```

Comment:
Shouldn't it be `EmbarkStory.SwedenQuoteCartNeeder` a bit down this file that should have this prop?

Reply:
Wow, good catch! You are correct!